src/main/java/design/MazeDesignerController.java

```diff
@@ -0,0 +1,52 @@
+package design;
+
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+
+public class MazeDesignerController {
+    MazeDesignerInteractor md = new MazeDesignerInteractor();
+    MazeDesignerPresenter mdp = new MazeDesignerPresenter();
```

Comment:
I don't think you should be calling methods from the presenter in the controller. Let the interactor handle any interactions with the presenter. Also, I think instead of having all this code in the controller, you should move it to the interactor and have the interactor handle all the relevant method calls.

Reply:
I agree here as well, I think we want to avoid storing an instance of the presenter in the controller, and avoid having `outWallAttempt` and `updateMaze` directly invoking functions on `mdp` (the presenter). Having it invoke a function on the use case interactor which then updates the presenter would be better, 

If the controller talks directly to the presenter we lose the ability to swap out controllers and presenters since they are now dependent.